### PR TITLE
update can json for segment voltages

### PR DESCRIPTION
**Pull request by fregely**
## Changes

Added the segment voltages to the can_message json

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

**Review by jr1221 — CHANGES_REQUESTED** [review]
see main PR for changes.  Also please add the sim struct (see other messages).  Since the cells range between 2.5 and 4.2, please make the sim range that too.

File: cangen/can-messages/bms.json [hunk 1]
```diff
@@ -1429,6 +1429,106 @@
...
     "sim_freq": 700
   },
+  {
+    "id": "0x90",
+    "desc": "Segment Voltages",
+    "points": [
+      {
+        "size": 12,
+        "sim": {
+          "min": 2.5,
```
Comment (jr1221) [inline cangen/can-messages/bms.json:1439]: 62.5 to 105

Comment (fregely) [reply cangen/can-messages/bms.json:1439]: are you saying that the min is 62.5V and the Max is 105V? cause that seems wrong

**Review by jr1221 — APPROVED** [review]
feel free timmerge sorry forgot you were calculating average voltages.
